bg_atlasgen/validate_atlases.py

```diff
@@ -179,17 +184,50 @@ def validate_atlas(atlas_name, version, validation_functions):
 
     valid_atlases = []
     invalid_atlases = []
+    successful_validations = {}
+    failed_validations = {}
+
     for atlas_name, version in get_all_atlases_lastversions().items():
-        successful_validations, failed_validations = validate_atlas(
+        temp_successful_validations, temp_failed_validations = validate_atlas(
             atlas_name, version, all_validation_functions
         )
+
+        successful_validations.update(temp_successful_validations)
+        failed_validations.update(temp_failed_validations)
+
         for item in successful_validations:
             valid_atlases.append(item)
         for item in failed_validations:
             invalid_atlases.append(item)
 
-    print("Summary")
-    print("### Valid atlases ###")
-    print(valid_atlases)
-    print("### Invalid atlases ###")
-    print(invalid_atlases)
+    print("Validation has been completed")
+    print(
+        "Find successful_validations.json and failed_validations.json in ~/.brainglobe/atlases/validation/"
+    )
+
+    # Get the directory path
+    output_dir_path = str(get_brainglobe_dir() / "atlases/validation")
+
+    # Create the directory if it doesn't exist
+    if not os.path.exists(output_dir_path):
+        os.makedirs(output_dir_path)
+
+    # Open a file for writing
```

Comment:
```suggestion
    # Open a file for writing (will overwrite any files from previous runs!)
```
Worth being explicit here, I think?